redactrus_test.go

```diff
@@ -171,6 +171,25 @@ func TestStringer(t *testing.T) {
 	assert.Equal(t, logrus.Fields{"Stringer": "[REDACTED] is fmt.Stringer"}, logEntry.Data)
 }
 
+type S struct {
+	v string
+}
+
+func (s S) String() string {
+	return s.v
+}
```

Comment:
There is already a [Stringer struct defined in this file](https://github.com/whuang8/redactrus/blob/master/redactrus_test.go#L155-L161) I think we should reuse in your test.

Reply:
Ah yeah, totally. Going to merge this test with the existing stringer test then as well